cpu/efm32/periph/uart.c

```diff
@@ -68,7 +68,7 @@ int uart_init(uart_t dev, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
 
     /* initialize the pins */
     gpio_init(uart_config[dev].rx_pin, GPIO_IN_PU);
-    gpio_init(uart_config[dev].tx_pin, GPIO_OUT);
+    gpio_init(uart_config[dev].tx_pin, GPIO_OUT | 1); /* 1 for high */
```

Comment:
Would it be an option to invoke `gpio_set(uart_config[dev].tx_pin)` afterwards? That will also set the pin.

In the I2c driver, I do something [similar](https://github.com/RIOT-OS/RIOT/blob/master/cpu/efm32/periph/i2c.c#L114) (but for a different reason).

Reply:
If we call `gpio_set()` first, it gets overridden by `gpio_init()`. And if we call it after, we still get an unwanted TX assertion even though the duration is much shorter.